spec/actions/pod_push_with_error_handling.rb

```diff
@@ -1,12 +1,16 @@
 module Fastlane
   module Actions
-    
     class PodPushWithErrorHandlingAction < Action
```

Comment:
Same thing as in the first PR. Might be good to add some tests for the logic here.